### ISSUE TITLE
Update Docker Hub description using README.md

### ISSUE THREAD

**Issue by ArmaanT** [CircleCI-Public/docker-orb#46]
When publishing an image to Docker Hub, the description of the image doesn't update. There exists an (unofficial) way of updating the image description by sending a PATCH request to the page of the repository on Docker Hub.

I would be willing to create a PR to set up this functionality but would like feedback on whether or not this would be a useful addition. My tentative idea is to provide an additional command that updates the description from a README file and that command is then optionally called in the publish job.

**Comment by iynere** [comment]
@ArmaanT yes, totally, that would be awesome! we would have definite use for that internally, as well. thank you!
